dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/MacCatalystApp1.csproj

```diff
@@ -1,11 +1,33 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>net7.0-maccatalyst</TargetFramework>
-    <RuntimeIdentifier>maccatalyst-x64</RuntimeIdentifier>
+  <PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst')) and '$(Configuration)' == 'Debug'">
+		<MtouchLink>none</MtouchLink>
+		<EnableCodeSigning>false</EnableCodeSigning>
+		<ProvisionType>Manual</ProvisionType>
+		<CreatePackage>false</CreatePackage>
+		<EnablePackageSigning>true</EnablePackageSigning>
+		<CodesignKey>Apple Distribution: YOURNAME (*******)</CodesignKey>
+		<CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
+		<CodesignProvision>YOUR PROFILE NAME</CodesignProvision>
+		<PackageSigningKey>3rd Party Mac Developer Installer: YOURNAME (*******)</PackageSigningKey>
+		<UseHardenedRuntime>false</UseHardenedRuntime>
+	</PropertyGroup>
+	<PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst')) and '$(Configuration)' == 'Release'">
+		<MtouchLink>Full</MtouchLink>
+    <--You must specify both both architectures or only x64 as desired. arm64 only is not permitted.-->
+		<RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers>
```

Comment:
since x64 has to be included, maybe you could leave that as the default identifier and add a comment about adding arm64 if wanted?